### ISSUE TITLE
Bot suffocating from using Oxygenite Shard in their diving suit

### ISSUE THREAD

**Issue by LemADEC** [Regalis11/Barotrauma#7910]
- [x] I have searched the issue tracker to check if the issue has already been reported.

**Description**
Bot used an Oxygenite Shard in their diving suit then died from suffocation.

**Steps To Reproduce**
1- Equip bot with PUCS and an oxygen tank
2- Order bot to repair hull, keep oxygenite shard in various storage, leave oxygen tank in center
3- Aggro a swarm to get continuous hull damage
=> bot tries to repair hull, once they're out of oxygen, they replace their tank with Oxygenite shard instead of Oxygen tank

**Version**
0.15.22.1 Windows

**Additional information**
I had a few bots dying from suffocation, couldn't understand until I've checked their PUCS and found a shard in there.
Issue was seen in a custom sub with no mods, playing alone in a multiplayer instance.

**Expected behavior**
Bot using valid oxygen source in their equipment.

**Comment by itchyOwl** [comment]
Can't repro. Oxygen shards are no longer oxygen sources and can't be used in the suits since 0.15.22.

I tried to make it happen by equipping the PUCS suit for a bot, getting rid of all the oxygen tanks, spawning oxygenite shards everywhere and keeping the sub flooded. The bot did not put the shards in the suit and ended up dying each time I tested. Just as expected. I wonder what's going on there...

**Comment by Oukido** [comment]
Was able to repro:
![grafik](https://user-images.githubusercontent.com/83960267/148081634-c760b4da-65da-45ab-889b-f87411152b25.png)

Additional info:
I took the bottle out of the bots suit and it instantly ran to the canister rack. In the "loose vent" beside the rack I had stored an oxigenite shard from pre patch which still contained the tag.

This means every oxygenite shard placed in a sub pre patch still contains the tag and can be put into the tank slot similar to the fire extinguisher trick for custom display shelfes.

**Comment by itchyOwl** [comment]
Right, I suspected something like that, but couldn't get my brain in the correct orientation to trace it like you did! Thanks! Fixed in Regalis11/Barotrauma-development@769d0ff.

**Comment by itchyOwl** [comment]
As a further notice: it's not really an issue with the bots. The player can also put the shards from an old save into a suit, because the shards are tagged as "oxygensource". Hence, I'll update the title.

**Comment by Rokvach** [comment]
Tested, working correctly. Closing.
